packages/salesforcedx-vscode-replay-debugger/src/index.ts

```diff
@@ -86,21 +77,43 @@ export async function activate(context: vscode.ExtensionContext) {
   }
   console.log('in activate, successfully retrieved line breakpoint info');
 
-  console.log('in activate, activating request service');
-  if (!await checkpointService.activateRequestService()) {
-    console.log('in activate, activating request service failed, returning');
-    // the error was already reported in the function, return
-    return;
-  }
-  console.log('in activate, successfully activated request service');
-
-  const breakpointsSub = vscode.debug.onDidChangeBreakpoints(
-    processBreakpointChangedForCheckpoints
+  const config = vscode.workspace.getConfiguration();
+  const checkpointsEnabled = config.get(
+    'salesforcedx-vscode-replay-debugger-checkpoints.enabled',
+    false /* value to return if one is not defined */
```

Comment:
This doesn't need a comment. Hovering over `config.get()` already says the second param is the default value.